### ISSUE TITLE
Problem upgrading zigbee2mqtt edge.

### ISSUE THREAD

**Issue by ritehaus** [zigbee2mqtt/hassio-zigbee2mqtt#96]
I have upgraded zigbee2mqtt edge add-on on Home Assistant. I copied/pasted my configuration as described but when I reopened the add-on UI. all devices had disappeared and I am having to manually re-add them 1 by 1

How do I upgrade and retain all existing devices in future?
```

**Comment by ciotlosm** [comment]
You should not be losing your devices. If you use edge from same repo as the normal one, the only tying to do is to stop the stable and start edge and it should work. I don't remember there being any breaking changes in between them for a while.

**Comment by ciotlosm** [comment]
I'll close this issue as i've extensively tested moving from stable to edge and updating edge with no devices lost. The only explanation is when we switched repos if you someone didn't get the config moved in time before starting the new edge for the 1st time. 

Please open a new issue if you can't upgrade edge in the future without losing devices.
